public_html/package-stats.php

```diff
@@ -255,9 +255,9 @@ function updateGraph()
     }
 </script>
 <form name="graph_control" action="#">
- <input type="hidden" name="pid" value="<?php echo @$_GET['pid']; ?>" />
- <input type="hidden" name="rid" value="<?php echo @$_GET['rid']; ?>" />
- <input type="hidden" name="cid" value="<?php echo @$_GET['cid']; ?>" />
+ <input type="hidden" name="pid" value="<?php echo empty($_GET['pid']) ? '' : $_GET['pid']; ?>" />
+ <input type="hidden" name="rid" value="<?php echo empty($_GET['rid']) ? '' : $_GET['rid']; ?>" />
+ <input type="hidden" name="cid" value="<?php echo empty($_GET['cid']) ? '' : $_GET['cid']; ?>" />
```

Comment:
```suggestion
 <input type="hidden" name="cid" value="<?php echo isset($_GET['cid']) ? $_GET['cid'] : ''; ?>" />
```

Reply:
Because when $_GET['someid'] = 0 it will be also empty and it won't output zero...